packages/react-slider/src/components/Slider/Slider.stories.tsx

```diff
@@ -15,7 +15,15 @@ export default {
   component: Slider,
   decorators: [
     Story => (
-      <div style={{ display: 'flex', flexDirection: 'column', gap: '1em', alignItems: 'flex-start', padding: 20 }}>
+      <div
+        style={{
+          display: 'grid',
```

Comment:
Can you add a comment about what this is doing and why you did this instead of flex?

Reply:
@micahgodbolt  I believe just linking this PR should be enough of a comment as it is described in the description. :)

Reply:
Done

Reply:
Thanks! Yeah linking the PR for more explanation is good--it's of course also possible to find that from the history with a little work, but having it visible in a comment is better.